src/components/organisms/AdminVenueView/components/PrettyLink/PrettyLink.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
```

Comment:
Not sure about the name for this component
are there any differences except that it's pretty? We might need a better name that covers this link's difference from others

Reply:
@alexey-petrov FYI https://github.com/sparkletown/sparkle/pull/1705#discussion_r685017684